.travis.yml

```diff
@@ -48,7 +53,11 @@ deploy:
     repo: nokia/git-changelog-generator
 - provider: script
   skip_cleanup: true
-  script: bash scripts/bintray/deploy.bash "$BINTRAY_USER" "$BINTRAY_KEY" "$(< version.txt)"
+  script: >
+            DEBUG=1 bash scripts/bintray/deploy.bash -u "$BINTRAY_USER" \
+              -p "$BINTRAY_KEY" -v "$(< version.txt)" \
+              -r ${lifecycle:-${DEFAULT_RPM_LIFECYCLE}} \
+              -c ${component:-${DEFAULT_DEB_COMPONENT}} -d 
```

Comment:
Is DEFAULT_DEB_COMPONENT right here? (not RPM?) If it is right, should DEB be in the name?

Reply:
I believe it's right.
`deploy.sh` is a wrapper to deploy both RPMs and DEBs both because the Travis deployment provider doesn't like to run multiple commands (which is IMVHO silly but who am I to judge that).

So the parameters are: 
- Component for Debian (as there's just one repository), e.g. dev/main/testing...
- an artificial "lifecycle" which basically points to a dev/main/testing Yum repo.

Doe this make sense?

Reply:
My comment doesn't make sense to me any more. On the other hand your explanation does,